cmd/istio-cni/main.go

```diff
@@ -166,12 +168,18 @@ func cmdAdd(args *skel.CmdArgs) error {
 			if err != nil {
 				return err
 			}
-			log.Debug("Created Kubernetes client",
-				zap.Reflect("client", client))
-			containers, _, annotations, ports, k8sErr := getKubePodInfo(client, string(k8sArgs.K8S_POD_NAME), string(k8sArgs.K8S_POD_NAMESPACE))
+			log.Debug("Created Kubernetes client", zap.Reflect("client", client))
+			containers, initContainersMap, _, annotations, ports, k8sErr := getKubePodInfo(client, string(k8sArgs.K8S_POD_NAME), string(k8sArgs.K8S_POD_NAMESPACE))
 			if k8sErr != nil {
 				log.Warnf("Error getting Pod data %v", k8sErr)
 			}
+
+			// Check if istio-init container is present; in that case exclude pod
+			if _, present := initContainersMap[ISTIOINIT]; present {
+				log.Infof("Found istio-init container for pod: %s; namespace: %s", string(k8sArgs.K8S_POD_NAME), string(k8sArgs.K8S_POD_NAMESPACE))
```

Comment:
Make it clear in the log that the pod is excluded, following a pattern similar to the one below:
```suggestion
				log.Info("Pod excluded due to being already injected with istio-init container",
					zap.String("pod", k8sArgs.K8S_POD_NAME),
					zap.String("namespace", k8sArgs.K8S_POD_NAMESPACE))
```

Reply:
done